bin/MadGraph5_aMCatNLO/cards/VLT_ta/VLT_ta_M300/VLT_ta_M300_customize.dat

```diff
@@ -0,0 +1,2 @@
+set param_card Mu4 300 
+set param_card MHDO 1000
```

Comment:
I see only uq4 appears in the syntax, will the "MHD0" affect the process

Reply:
The MHDO parameter affects the cross-section value.